### PR TITLE
Update samples and their docs

### PR DESCRIPTION
**Pull request by jngrad**
Fixes issues found while setting up the samples for the *Tag der Wissenschaft* 2019.

### CODE REVIEW

File: samples/lj-demo.py [hunk 1]
```diff
@@ -182,6 +183,9 @@ class Controls(HasTraits):
     default_output = default_output if len(
         outputs) > 1 else through_port_output
 
+    if default_input is None or default_output is None:
+        raise RuntimeError('Cannot connect to any MIDI device')
```
Comment (mkuron) [inline samples/lj-demo.py]: This shouldn't raise an exception as someone might want to run the demo without a MIDI controller. Instead, just print a message.

**Review by mkuron — CHANGES_REQUESTED** [review]

File: samples/lj-demo.py (hunk 1 above)
Comment (jngrad) [reply samples/lj-demo.py]: done

**Review by RudolfWeeber — APPROVED** [review]

**Comment by jngrad** [comment]
bors retry

**Comment by jngrad** [comment]
@mkuron we're not sure what is the issue here, could you please change your review from "Changes requested" to "Approved" to see if bors is picking up your unfinished review?

**Review by mkuron — APPROVED** [review]

**Comment by jngrad** [comment]
bors retry

**Comment by jngrad** [comment]
@mkuron thanks!
@fweik @KaiSzuttor to keep in mind: bors will refuse a merge of an approved PR when another (possibly non-admin) contributor requested a change, even if that change was made and marked as resolved, until the contributor making the request updates the review to "approved"

**Pull request merged**